File: DS-Algo-with-Java/Lecture-7/PyramidPattern.java
import java.util.Scanner;

public class PyramidPattern {
    public static void main(String[] args) {
        
        try (Scanner sc = new Scanner(System.in)) {

            System.out.print("Enter the height: ");
            int height = sc.nextInt();

            for (int i = 1; i <= height; i++) {

                for (int k = 0; k < height - i; k++) {
                    System.out.print(" ");
                }

                for (int j = 1; j <= (2 * i) - 1; j++) {
                    System.out.print("*");
                }

                System.out.println();
            }
        }
    }
}

File: DS-Algo-with-Python/Lecture-7/PyramidPattern.py
height = int(input("Enter the height: "))

for i in range(1, height + 1):
    for k in range(0, height - i):
        print(" ", end="")
    
    for j in range(1, (2 * i)):
        print("*", end="")
    
    print("")
